entity-service/src/main/kotlin/com/egm/stellio/entity/util/QueryUtils.kt

```diff
@@ -161,13 +161,22 @@ object QueryUtils {
                     parsedQueryTerm.third.isTime() -> "localtime('${parsedQueryTerm.third}')"
                     else -> parsedQueryTerm.third
                 }
-                """
+                if (parsedQueryTerm.first.endsWith(".observedAt"))
+                    """
+                   EXISTS {
+                       MATCH (n)-[:HAS_VALUE]->(p:Property)
+                       WHERE p.name = '${parsedQueryTerm.first.removeSuffix(".observedAt")}'
+                       AND p.observedAt ${parsedQueryTerm.second} $comparableValue
+                   }
+                    """.trimIndent()
```

Comment:
It seems it would not be too difficult to have something more generic?

For instance, by splitting on the '.' character and use the second part on L.169.